packages/apps-config/src/endpoints/index.ts

```diff
@@ -31,7 +31,16 @@ export function createWsEndpoints (t: TFunction, firstOnly?: boolean): LinkOptio
       textBy: '',
       value: ''
     },
-    ...createTestingRelays(t, firstOnly),
+    ...createRococoRelays(t, firstOnly),
+    {
+      isDisabled: false,
+      isHeader: true,
+      isSpaced: true,
+      text: t('rpc.header.live', 'Live networks', { ns: 'apps-config' }),
```

Comment:
```suggestion
      text: t('rpc.header.westend.relay', 'Westend test & parachains', { ns: 'apps-config' }),
```

Would just appreciate an update on line 30 as well to `Rococo test & parachains` (Sadly cannot get to it via the GH editor)

Thanks fo r this split, it was meant to be done when Polkadot has at least parathreads.

Reply:
Done